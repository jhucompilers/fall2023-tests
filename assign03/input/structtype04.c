struct Item {
  int foo;
  char bar;
};

int main(void) {
  struct Item x, *y, z[10];

  y = &x;
  y->foo = 17;
  y->bar = '!';
  z[0].foo = 42;
  z[0].bar = '*';

  return 0;
}
