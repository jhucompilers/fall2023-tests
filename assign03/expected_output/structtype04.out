0|struct Item|type|struct Item {}
1|foo|variable|int
1|bar|variable|char
0|main|function|function () returning int
2|x|variable|struct Item {int, char}
2|y|variable|pointer to struct Item {int, char}
2|z|variable|array of 10 x struct Item {int, char}
